quipucords/scanner/normalizer.py

```diff
@@ -112,23 +120,31 @@ class FactMapper:
 
     def __init__(
         self,
-        raw_fact_key: str,
+        raw_fact: str | list[str] | None,
         normalizer: callable,
         *,
         validators=None,
         dependencies: list[str] = None,
     ) -> None:
         """Instantiate FactMapper.
 
-        :param raw_fact_key: name of the raw fact collected during inspection phase
-        :param normalizer: function used to normalize raw fact (signature MUST be
-            fn(raw_fact_value, **dependencies) where dependencies are kwargs from
-            dependencies)
+        :param raw_fact: raw fact name or list of raw fact names used to normalize a
```

Comment:
I prefer the old name or something like it. Just calling this "raw fact" implies that it is the fact itself, not the name of the fact, and that's an anti-pattern that has caused me confusion elsewhere.

Reply:
OK! I shall keep it as `raw_fact_key`

Reply:
Also, mixing types here feels weird to me. If we expect this to sometimes handle a list of strings, I'd make it always expect a list of strings, even if it's often a list of length one. We don't have to change this; just noting an odd feeling.

Reply:
…aaaand after reading a later commit that adds lots of FactMapper instantiations, I see why you made this type flexible since lists are in the minority. I guess I wouldn't want to make them all lists for no other reason.